Adyen/Components/AbstractPersonalInformationComponent/FormItemInjector.swift

```diff
@@ -12,4 +12,26 @@ public protocol FormItemInjector {
 
     /// :nodoc:
     func inject(into formViewController: FormViewController)
+    
+}
+
+/// Injects a custom `FormItem` into a `FormViewController`.
+/// :nodoc
+public struct CustomFormItemInjector<T: FormItem>: FormItemInjector {
```

Comment:
This could be a base class for all FormItemInjector ?

Reply:
no need for that, we have the protocol for this, its provides no value.

Reply:
i'm not sure it is possible due to the combination of generics and protocol